src/primitives/transaction.cpp

```diff
@@ -96,18 +96,9 @@ class SproutProofVerifier : public boost::static_visitor<bool>
 
     bool operator()(const libzcash::PHGRProof& proof) const
     {
-        return params.verify(
```

Comment:
Removing this causing some error in btest. Please help to check.

Reply:
I can be wrong, but if we don't need JoinSplit transactions, we can remove SproutProofVerifier, JSDescription classes.

Reply:
Yes, I found the issue 3 from 4 tests I've fixed. I'm trying to make it with small changes. Otherwise we should totally remove libsnark that will have large impact.